ci-operator/step-registry/rhcs/e2e/setup/rhcs-e2e-setup-ref.metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+	"path": "rhcs/e2e/setup/rhcs-e2e-setup-ref.yaml",
+	"owners": {
+		"approvers": [
+			"bardielle",
+			"gdbranco",
+			"sagidayan",
+			"vkareh",
+			"osherdp",
+			"AsherShoshan",
+			"nirarg",
+			"xueli",
+			"yasun",
+			"yufchang",
+			"amalykhi"
+		],
+		"reviewers": [
+			"bardielle",
+			"gdbranco",
+			"sagidayan",
+			"vkareh",
+			"osherdp",
+			"AsherShoshan",
+			"nirarg",
+			"xueli",
+			"yasun",
+			"yufchang",
+			"yuwan",
+			"yingzhang",
+			"tzhou",
+			"mgahagan",
+			"yisun",
+			"amalykhi",
+			"smiron"
```

Comment:
my username on GitHub is `shellymiron` - does it matter here?

Reply:
I found usernames issue, bug some guys not online on weekend, fixing now.